src/hotspot/share/runtime/arguments.cpp

```diff
@@ -4188,6 +4188,16 @@ jint Arguments::parse(const JavaVMInitArgs* initial_cmd_args) {
     PropertyList_add(&_system_properties, new SystemProperty("java.math.BigDecimal.optimization", "true", true));
   }
 
+  if (PolymorphicInlining) {
+    FLAG_SET_ERGO_IF_DEFAULT(intx, TypeProfileWidth, 6);
```

Comment:
We can set TypeProfileWidth = 8 for more accurate profiling.

Reply:
done